Syntaxes/Julia.tmLanguage

```diff
@@ -244,13 +244,13 @@
 			<array>
 				<dict>
 					<key>match</key>
-					<string>\b(?:function|stagedfunction|type|immutable|macro|quote|abstract|bitstype|typealias|module|baremodule|new)\b</string>
```

Comment:
`new` can be used as just a regular old variable name though outside of type context - unless the grammar highlighting is context sensitive enough to do it right, it's not the same as other keywords